### PR TITLE
Update Readme files and some screenshots to reflect data index servic…

### PR DESCRIPTION
**Pull request by nmirasch**
…e stuff. Updated some notes

@mswiderski  @cristianonicolai Starting point to improve tutorial readme files. Please, read carefully and make suggestions. @cristianonicolai I wasn't sure about include that .proto files to start the service regarding they are already at kogito-runtimes repo but if the tutorial have to be an starting point that have to be done with any new model.. anyway.. any comment is very welcome! Thanks!

### CODE REVIEW

File: 04-kogito-travel-agency/README.md [hunk 1]
```diff
@@ -158,6 +158,33 @@ To run the generated native executable, generated in `target/`, execute
 ./target/kogito-travel-agency-{version}-runner
 ```
 
+### Start Kogito Data Index Service
+
+ We have to start the Data Index Service, we can find this service description more detailed at [Kogito Data Index Service](https://github.com/kiegroup/kogito-runtimes/wiki/Data-Index-Service).
```
Comment (mswiderski) [inline 04-kogito-travel-agency/README.md:163]: I would actually point people to runner jar from maven - either central for released version or to jboss repo where we have snapshots

File: 04-kogito-visas/README.md [hunk 2]
```diff
@@ -113,6 +111,34 @@ To run the generated native executable, generated in `target/`, execute
 ./target/kogito-visas-{version}-runner
 ```
 
+### Start Kogito Data Index Service
+
+We have to start the Data Index Service, we can find this service description more detailed at [Kogito Data Index Service](https://github.com/kiegroup/kogito-runtimes/wiki/Data-Index-Service).
```
Comment (mswiderski) [inline 04-kogito-visas/README.md]: Same here point people to maven repo and run the data index ‘java -jar runner’ with the proto folder property

**Review by mswiderski — COMMENTED** [review]
Excellent write up, adres two comments about running data index. Other than that looks good to me

File: 03-kogito-travel-agency/README.md [hunk 3]
```diff
@@ -122,9 +123,11 @@ When using native image compilation, you will also need:
 
 #### Infinispan
 
-This application requires an Inifinispan server to be available and by default expects it to be on default port and localhost.
+This application requires an Infinispan server to be available and by default expects it to be on default port and localhost.
 
-You can install Inifinispan server by downloading it from [official website](https://infinispan.org/download) version to be used in 10.0.0.Beta4
+You can install Infinispan server by downloading it from [official website](https://infinispan.org/download) version to be used in 10.0.0.CR1
```
Comment (cristianonicolai) [inline 03-kogito-travel-agency/README.md]: @nmirasch I've been using CR2, could we update to that?

File: 04-kogito-travel-agency/README.md (hunk 1 above)
Comment (cristianonicolai) [reply 04-kogito-travel-agency/README.md:163]: +1

**Review by cristianonicolai — COMMENTED** [review]
@nmirasch looks great thanks, just some minor comments.

**Comment by nmirasch** [comment]
@mswiderski @cristianonicolai updated the way to start the data-index using runners and not depend on snapshots. Could you take another look? Thanks!!

File: 04-kogito-travel-agency/README.md [hunk 4]
```diff
@@ -158,6 +158,28 @@ To run the generated native executable, generated in `target/`, execute
...
+  
+ It can be downloaded from [Kogito Dada Index Service 0.4.0](http://repo2.maven.org/maven2/org/kie/kogito/data-index-service/0.4.0/data-index-service-0.4.0-runner.jar)
```
Comment (mswiderski) [inline 04-kogito-travel-agency/README.md:165]: maybe just add additional note so people can find other versions as well 

````
NOTE: Here can be found other versions [Kogito Dada Index Service versions](http://repo2.maven.org/maven2/org/kie/kogito/data-index-service)
````

**Review by mswiderski — APPROVED** [review]
added minor comment but I'd say it's optional, decide yourself if it makes sense :)

File: 03-kogito-travel-agency/README.md (hunk 3 above)
Comment (nmirasch) [reply 03-kogito-travel-agency/README.md]: updated!

File: 04-kogito-travel-agency/README.md (hunk 4 above)
Comment (nmirasch) [reply 04-kogito-travel-agency/README.md:165]: sure, it makes sense, added!

**Review by cristianonicolai — APPROVED** [review]
@nmirasch nice, thanks for this update!

**Pull request merged**